imports/client/components/App.tsx

```diff
@@ -239,12 +250,13 @@ const AppNavbar = () => {
   // Note: the .brand class on the <img> ensures that the logo takes up the
   // correct amount of space in the top bar even if we haven't actually picked
   // a nonempty source for it yet.
+
+  const theme = useTheme();
   return (
     <NavbarInset
       variant="light"
       style={{
-        backgroundColor: "#f0f0f0",
```

Comment:
any reason you dropped the `background-color` on the navbar here rather than theming it?